.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for this project
+
+---
+
+/kind feature
+
```

Comment:
i suggest to add a `use case` section, this is important to understand why people want this feature.

Reply:
Hi @figo,

That's great feedback. I'm going to say what I said above - how about we get this PR merged so we're in sync with community standards and then we can enhance the templates later?